internal/scope/local.go

```diff
@@ -32,14 +32,18 @@ func NewLocal(projectConfigFile string, projectInstallDir string, noInit bool, n
 	return &local{projectConfigFile: projectConfigFile, installDir: projectInstallDir, noInit: noInit, noSave: noSave, os: afero.NewOsFs()}
 }
 
-func (l *local) ValidatePaths() error {
+func (l *local) ValidatePaths(ctx *context.CLIContext) error {
 	currentUser, err := user.Current()
 	if err != nil {
 		return fmt.Errorf("can't determine current user")
 	}
 
 	projectDir := path.Dir(l.projectConfigFile)
 
+	if projectDir == ctx.Paths.GlobalInstallDir {
```

Comment:
This is very much an edge-case - not sure if necessary. It requires passing context just for this single thing.

Reply:
I guess you are right - this is an edge case and from what I tested just now it does not cause any major issues 
It treats the global install dir just like any other project dir and it does not interfere with globally installed commands 

I will remove this condition and won't pass `ctx`

Reply:
condition removed, `ctx` is no longer passed